motion/ruby_motion_query/utils.rb

```diff
@@ -118,6 +118,22 @@ def project_path
 
           ppath
         end
+
+        # To use this, put this in your Rakefile:
+        #   app.development do
+        #     app.info_plist["ProjectBuildTime"] = Time.now
+        #   end
+        def build_time
+          if btime = NSBundle.mainBundle.infoDictionary["ProjectBuildTime"]
```

Comment:
Yeah, that's what I thought... but I was copying the implementation of `project_path` in this file. Thought there might be some convention there.
https://github.com/infinitered/rmq/blob/2ac7dff1753e1b17e10fa3617621d67a189609b3/motion/ruby_motion_query/utils.rb#L109-L120